packages/circuits/tests/poseidon-cipher.test.ts

```diff
@@ -1,13 +1,13 @@
 import { WitnessTester } from "circomkit"
 import { Nonce, PlainText, poseidonDecrypt, poseidonEncrypt, poseidonPerm } from "@zk-kit/poseidon-cipher"
-
-import { circomkit, genEcdhSharedKey, genPublicKey, genRandomBabyJubValue } from "./common"
+import { cryptoNode } from "@zk-kit/utils"
```

Comment:
It would better to use selective imports when possible, i.e. "@zk-kit/utils/crypto"